pmg/templates/admin/my_edit.html

```diff
@@ -12,6 +12,40 @@
     </div>
     {% endif %}
 
+    {% if admin_view.name == 'Users' %}
+
+    <button type="button" class="btn btn-danger btn-sm pull-right" data-toggle="modal" data-target="#password-reset-modal">
+      Reset user password
+    </button>
+
+    <div class="modal fade" id="password-reset-modal" tabindex="-1" role="dialog" aria-labelledby="password-reset-label">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                    <h4 class="modal-title" id="password-reset-label">Reset user password</h4>
+                </div>
+                    <form action="{{ get_url('.reset_user_password', model_id=model.id, url=return_url) }}" method="POST" class="admin-form form-horizontal" enctype="multipart/form-data">
+                    <div class="modal-body">
+                        <p>Please enter a new password. It should contain at least 6 characters, and no spaces.</p>
```

Comment:
I think we should mention that the admin MUST tell the user what their new password is. Otherwise the flash message that says "and a notification has been sent" might confuse them into thinking the new password has been emailed to them.